docs/sources/developers/http_api/annotations.md

```diff
@@ -121,6 +121,12 @@ See note in the [introduction]({{< ref "#annotations-api" >}}) for an explanatio
 | ------------------ | ----------------------- |
 | annotations:create | annotations:type:<type> |
 
+
+**Required JSON Body Fields**
+  - `time`: epoch datetime in milliseconds.
+  - `timeEnd`: epoch datetime in milliseconds - must be greater than `time`.
```

Comment:
I don't think that those two fields are required
I can successfully create an annotation without them:
```
curl -X 'POST'   'http://<user>:<password>@localhost:3000/api/annotations'   -H 'accept: application/json'   -H 'Content-Type: application/json'   -d '{
  "dashboardId": 0,
  "dashboardUID": "string",
  "data": {},
  "panelId": 0,
  "tags": [
    "string"
  ],
  "text": "string"
}'
{"id":37712,"message":"Annotation added"}
```

Reply:
Ah, I was getting an unrelated error when trying to add without those fields - I'll remove them from the change request. Thanks for the sanity check